hack/verify-staticcheck.sh

```diff
@@ -39,4 +39,4 @@ else
   echo 'Please review the above warnings.'
   echo 'If the above warnings do not make sense, feel free to file an issue.'
   exit 1
-fi
+fi
```

Comment:
The commit message seem misleading, please reorganize your commit

Reply:
done